package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@chainsafe/dappeteer",
-  "version": "2.4.0",
+  "version": "2.4.1",
```

Comment:
could you revert back version?

We would like to put in more bugfixes and this is new feature so we would need to bump minor version.

Reply:
Reverted, I hope everything else is in order.